job-server-extras/test/spark.jobserver/NamedObjectsJobSpec.scala

```diff
@@ -0,0 +1,113 @@
+package spark.jobserver
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import akka.testkit.{ ImplicitSender, TestKit }
+import com.typesafe.config.{ Config, ConfigFactory, ConfigValueFactory }
+import akka.testkit.TestProbe
+import spark.jobserver.CommonMessages.{ JobErroredOut, JobResult }
+import spark.jobserver.io.JobDAOActor
+import collection.JavaConversions._
+
+class NamedObjectsJobSpec extends JobSpecBase(JobManagerSpec.getNewSystem) {
+
+  private val emptyConfig = ConfigFactory.parseString("spark.jobserver.named-object-creation-timeout = 60 s")
+
+  before {
+    dao = new InMemoryDAO
+    daoActor = system.actorOf(JobDAOActor.props(dao))
+    manager = system.actorOf(JobManagerActor.props(JobManagerSpec.getContextConfig(adhoc = false)))
+    supervisor = TestProbe().ref
+  }
+
+  val jobName = "spark.jobserver.NamedObjectsTestJob"
+
+  describe("NamedObjects (RDD)") {
+    it("should survive from one job to another one") {
+
+      manager ! JobManagerActor.Initialize(daoActor, None)
+      expectMsgClass(classOf[JobManagerActor.Initialized])
+
+      uploadTestJar()
+      manager ! JobManagerActor.StartJob("demo", jobName, emptyConfig.withValue(NamedObjectsTestJobConfig.CREATE_DF, ConfigValueFactory.fromAnyRef(false))
```

Comment:
Replace the `emptyConfig.withValue(...)` stuff with `ConfigFactory.parseString(....)`, which is much more readable, and you could avoid the long line as well.